src/hooks/useContentManager.js

```diff
@@ -179,11 +176,13 @@ const useContentManager = () => {
         console.error("error :>> ", err);
       });
   }, []);
-
+  const pageConfigFormProps = useMemo(() => {
+    return { pageConfig: _.cloneDeep(pageConfig), handleChange, errors: _.cloneDeep(errors) };
+  }, [errors, handleChange, pageConfig]);
   const dialogFormProps = useMemo(() => {
     return {
       0: {
-        dialogProps: { ...pageConfigFormProps, pageConfig: { ...pageConfig } },
+        dialogProps: { ...pageConfigFormProps, pageConfig: _.cloneDeep(pageConfig) },
```

Comment:
Using lodash for just this is not recommended instead you can try some other alternates like , if the dialogProps is not performing any mutation then just use pageConfig as it is or you can also use 
```javascript
JSON.parse(JSON.stringify(object))
```